### DISCUSSION TITLE
some files causes openwebui to close without error

### DISCUSSION THREAD

**Discussion by tcgumus** [open-webui/open-webui#4892] [General]
# Bug Report

## Installation Method

i use git clone and everything works until someone loads document with big table in it or big csv document. I can't find any error on logs even in debug mode. 

## Environment

- **Open WebUI Version:** [e.g., v0.3.11]
only using chatgpt

- **Operating System:** windows server 2019, pipeline works on docker i connect it succesfully with port. there is no rag customization. default settings is causing error also.

- **Browser (if applicable):** all

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.
- [x] I have included the Docker container logs.

## Actual Behavior:

i tried tika, changed extraction to chatgpt but still having the same issue. file is uploading correctly but when i submit it, it breaks the program and causes sudden shut down of service. 

## Description

**Bug Summary:**

Everything works until someone loads document with big table in it or big csv document. I can't find any error on logs even in debug mode. 

i tried tika, changed extraction to chatgpt but still having the same issue. file is uploading correctly but when i submit it, it breaks the program and causes sudden shut down of service. 

could you lead me to detailed logs of RAG or any environment variable or way to close big document uploads or file uploads all together. I can'e debug this error :/

there is no customization on the platform.

**Comment by tcgumus** [comment]
After weeks of struggling to find a solution, I finally gave in and opened an issue on GitHub. And guess what? Less than an hour later, I might have found the solution! 🎉 It seems like the problem was related to a bug involving uvicorn and asyncio on Windows, specifically in production with SSL.

I've got a long testing journey ahead, but right now, it looks like things are working. I’d like to leave this issue open for about a week, as I think it could help others facing similar problems. I’m actively working on it and will share my test results soon. A PR is also in the works!

Thanks for your patience!

**Reply by pkaramertzanis** [reply]
I have the same issue. What is the solution?

**Reply by adarshdotexe** [reply]
Same issue for me as well, please tell steps to fix, facing in prod on linux using python and milvus
